spring-cloud-aws-sqs/src/main/java/io/awspring/cloud/sqs/listener/SqsContainerOptions.java

```diff
@@ -181,6 +196,13 @@ public SqsContainerOptionsBuilder messageVisibility(Duration messageVisibility)
 			return this;
 		}
 
+		@Override
+		public SqsContainerOptionsBuilder batchGroupingStrategy(FifoBatchGroupingStrategy batchGroupingStrategy) {
```

Comment:
I think we can name this `fifoBatchGroupingStrategy`, since this only applies to FIFO queues, wdyt?

Reply:
I agree!